database/database.go

```diff
@@ -176,6 +182,62 @@ func (d *Database) PostPost(post *model.Post) (postid string, err error) {
 	return id, nil
 }
 
+func (d *Database) DeleteThread(threadId string) (err error) {
+	sqlStatement := `
+		DELETE FROM board.thread
+		WHERE Id = $1`
+	res, err := DB.Exec(sqlStatement, threadId)
+	if err != nil {
+		panic(err)
+	}
+	count, err := res.RowsAffected()
+	if err != nil {
+		panic(err)
+	}
+
+	if (count > 0) {
+		return
+	}
+
+	return errors.New("Couldn't find that thread")
+}
+
+func (d *Database) EditPost(editPost *model.Post) (err error) {
+	sqlStatement := `
+		UPDATE board.thread_post
```

Comment:
As a part of updating, it might be a good thing to have a column like EditedAt, and the UserId that edited it. This would be good traceability for if Mods or Admins edit a post later, so whoever is running the board can see who did it, and when.

Reply:
Cool, yeah, that makes sense. I'll add those in.